pandas/tests/plotting/common.py

```diff
@@ -598,3 +598,16 @@ def _gen_two_subplots(f, fig, **kwargs):
     else:
         kwargs["ax"] = fig.add_subplot(212)
     yield f(**kwargs)
+
+
+try:
+    import matplotlib as mpl
```

Comment:
You probably want to add 3.6.0 to https://github.com/pandas-dev/pandas/blob/main/pandas/plotting/_matplotlib/compat.py#L20-L21 instead.

Reply:
Thx, changed